src/Symfony/Component/HttpKernel/DataCollector/EventDataCollector.php

```diff
@@ -64,50 +66,74 @@ public function lateCollect()
      *
      * @param array $listeners An array of called listeners
      *
-     * @see TraceableEventDispatcherInterface
+     * @see TraceableEventDispatcher
      */
     public function setCalledListeners(array $listeners)
     {
         $this->data['called_listeners'] = $listeners;
     }
 
     /**
-     * Gets the called listeners.
```

Comment:
the diff is kinda messy... cant we avoid it by simply appending set+get (which seem to be the pattern anyway).